wandb/cli/cli.py

```diff
@@ -222,9 +222,16 @@ def projects(entity, display=True):
     "--relogin", default=None, is_flag=True, help="Force relogin if already logged in."
 )
 @click.option("--anonymously", default=False, is_flag=True, help="Log in anonymously")
-@click.option("--verify", default=False, is_flag=True, help="Verify login credentials")
+@click.option(
+    "--no-verify",
```

Comment:
you can do `"--verify/--no-verify"` see here for example: https://github.com/wandb/wandb/blob/main/wandb/cli/beta.py#L69